core/controlplane/config/templates/cloud-config-controller

```diff
@@ -4873,6 +4878,118 @@ write_files:
         namespace: kube-system
 {{end}}
 
+{{if .Experimental.GpuSupport.Enabled }}
+  - path: /srv/kubernetes/manifests/nvidia-driver-installer.yaml
+    content: |
+      apiVersion: extensions/v1beta1
+      kind: DaemonSet
+      metadata:
+        name: nvidia-driver-installer
+        namespace: kube-system
+        labels:
+          app: nvidia-driver-installer
+        annotations:
+          scheduler.alpha.kubernetes.io/critical-pod: ''
+      spec:
+        template:
+          metadata:
+            labels:
+              name: nvidia-driver-installer
+            annotations:
+              scheduler.alpha.kubernetes.io/critical-pod: ''
+          spec:
+            nodeSelector:
+              kube-aws.coreos.com/gpu: nvidia
```

Comment:
Completed untested but just my guess - We still need to tolerate all the taints here. Otherwise, the driver won't be installed on tainted nodes and therefore we can't make gpu nodes dedicated to some workloads.